### ISSUE TITLE
Determine Dot Colors When Selected, etc.

### ISSUE THREAD

**Issue by wgranger** [zooniverse/liberating-the-liberator#64]
@beckyrother

This is a minor UI need. What color will distinguish the following annotation dots:

- Previous clustered annotations
- User's individual annotations
- User's selected annotations

The image below is used for reference.

![screen shot 2017-09-08 at 10 47 37 am](https://user-images.githubusercontent.com/14099077/30219930-3f530792-9483-11e7-9c22-c043de54f39e.png)

**Comment by beckyrother** [comment]
I'm happy with these colors, as well as the blue you're using for your own previous annotations (see image).
![anti-slavery_manuscripts](https://user-images.githubusercontent.com/1392130/30562109-f438ea8a-9c82-11e7-816c-f1d8e3202920.png)

**Comment by beckyrother** [comment]
Are there other colors I'm forgetting?

**Comment by wgranger** [comment]
To make sure we've got this nailed out:

Your current and previous annotations: cyan
Selected clustered annotation: green
Unselected clustered annotation: red

Is there a difference between clustered annotations that are retired and not? Line retirement seems a bit iffy, but I know we talked about it. 

I say "iffy" because we would just be changing the color of a line depending on how many people had annotated and maybe disabling the dropdown from displaying on a click. A user would still be able to add an annotation on that line by making new marks if (s)he wants to.
